src/lib/shell/commands/Dns.cpp

```diff
@@ -42,23 +42,23 @@ class DnsShellResolverDelegate : public Dnssd::ResolverDelegate
 public:
     void OnNodeIdResolved(const Dnssd::ResolvedNodeData & nodeData) override
     {
-        streamer_printf(streamer_get(), "DNS resolve for " ChipLogFormatX64 "-" ChipLogFormatX64 " succeeded:\n",
+        streamer_printf(streamer_get(), "DNS resolve for " ChipLogFormatX64 "-" ChipLogFormatX64 " succeeded:\r\n",
```

Comment:
Why is it needed? Doesn't it depend on your terminal emulator? Maybe we should make `streamer_printf` apply a platform-specific new-line character if not....

Reply:
It might be terminal specific the need of adding it, but i don't see why adding it would cause an issue the other

instead of having it implicitly in some terminals it is explicit for all.
Do you see a case where if would act differently ?

Reply:
No, I was just curious if this was something required by your platform.